hazelcast/src/main/java/com/hazelcast/spi/impl/merge/AbstractSplitBrainMergePolicy.java

```diff
@@ -14,13 +14,13 @@
  * limitations under the License.
  */
 
-package com.hazelcast.spi.merge;
+package com.hazelcast.spi.impl.merge;
```

Comment:
Are you sure this class should be private? It will not be exposed to the end user to create its own splitbrain merge policy?

Reply:
It's just there for our own convenience. If the class is public we give a promise to maintain it in the future. Why should we do this, if it's not needed?

Reply:
It's also implementing `IdentifiedDataSerializable` with our `SplitBrainDataSerializerHook.F_ID`, so I guess a user will not be able to use it, since the hook cannot be extended.